src/contracts/Content/Form/Provider/GroupedContentFormFieldsProviderInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\ContentForms\Content\Form\Provider;
+
+interface GroupedContentFormFieldsProviderInterface
+{
+    /**
+     * @param \Symfony\Component\Form\FormInterface[] $fieldsDataForm
+     * @phpstan-return array<string, array<int, string>> Array of fieldGroupIdentifier grouped by fieldGroupName.
+     */
+    public function getGroupedFields(array $fieldsDataForm): array;
+}
+
+class_alias(GroupedContentFormFieldsProviderInterface::class, 'EzSystems\EzPlatformContentForms\Content\Form\Provider\GroupedContentFormFieldsProviderInterface');
```

Comment:
If it's an entirely new interface, then it does not need a BC compatibility aliasing. Or more accurately, it's an error to add one, because autoloader will not pick the name up properly, and it will depend on the order of autoloading calls whether alias works or not.